stytch/attributes.go

```diff
@@ -138,4 +138,11 @@ type Session struct {
 	ExpiresAt             string                  `json:"expires_at,omitempty"`
 	Attributes            Attributes              `json:"attributes,omitempty"`
 	AuthenticationFactors []*AuthenticationFactor `json:"authentication_factors,omitempty"`
+	CustomClaims          interface{}             `json:"custom_claims"`
+}
+
+type ClaimsWrapper struct {
+	Session struct {
+		Claims interface{} `json:"custom_claims"`
+	} `json:"session"`
 }
```

Comment:
I think this needs a doc comment because it's not obvious how to use it, especially if this is intended for consumer use.

Reply:
https://github.com/stytchauth/stytch-go/pull/77/commits/dfbb30b779e6f147f8826be224139e93d3315407